### PR TITLE
Adding licensing information

### PR DESCRIPTION
**Pull request by saadullah01**
As indicated in the issue: https://github.com/operate-first/ai-for-cloud-ops/issues/12, I have added the LICENSE file to the repo.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2019, Emre Ates
```
Comment (abyrne55) [inline LICENSE]:
```suggestion
Copyright (c) 2022, Operate First Contributors
```

**Review by abyrne55 — CHANGES_REQUESTED** [review]
Please change the name/year as shown, and then this is good to go

**Comment by saadullah01** [comment]
@abyrne55 I have updated the LICENSE file

**Review by mtoslalibu — APPROVED** [review]
Looks good!

**Comment by abyrne55** [comment]
/approve

Thanks @saadullah01!

**Review by abyrne55 — APPROVED** [review]

**Pull request merged**